gpt_index/docstore.py

```diff
@@ -68,18 +44,8 @@ def load_from_dict(
             elif doc_type == Node.get_type():
                 doc: Node = Node.from_dict(doc_dict)
             else:
-                if type_to_struct is None:
-                    raise ValueError(
-                        "type_to_struct must be provided if type is index struct."
-                    )
-                # try using IndexStructType to retrieve documents
-                if doc_type not in type_to_struct:
-                    raise ValueError(
-                        f"doc_type {doc_type} not found in type_to_struct. "
-                        "Make sure that it was registered in the index registry."
-                    )
-                doc = type_to_struct[doc_type].from_dict(doc_dict)
-                # doc = index_struct_cls.from_dict(doc_dict)
+                raise NotImplementedError()
```

Comment:
let's raise an explicit value error for now

Reply:
revised